templates/default/sv-kafka-run.erb

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 exec 2>&1
+exec chpst -u <%= node["kafka"]["user"] %> bash -c 'ulimit -n <%= node["kafka"]["filehandle_limit"] %>'
```

Comment:
Should have been more clear on this, the exec will take over the process immediately and won't execute the start kafka lines below, this can be changed to just `ulimit -n <%= node["kafka"]["filehandle_limit"] %>`.